File: Canary+CoreDataProperties.m
//
//  Canary+CoreDataProperties.m
//  mage-ios-sdk
//
//  Created by Dan Barela on 8/2/18.
//  Copyright © 2018 National Geospatial-Intelligence Agency. All rights reserved.
//

#import "Canary+CoreDataProperties.h"

@implementation Canary (CoreDataProperties)

+ (NSFetchRequest<Canary *> *)fetchRequest {
    return [[NSFetchRequest alloc] initWithEntityName:@"Canary"];
}

@dynamic launchDate;

@end

File: Persistence/Sources/Persistence/CoreDataObjects/Canary+CoreDataProperties.swift
//
//  Canary+CoreDataProperties.m
//  mage-ios-sdk
//
//  Created by Dan Barela on 8/2/18.
//  Copyright © 2018 National Geospatial-Intelligence Agency. All rights reserved.
//

import Foundation
import CoreData

public extension Canary {
    @nonobjc class func fetchRequest() -> NSFetchRequest<Canary> {
        return NSFetchRequest<Canary>(entityName: "Canary")
    }
    
    @NSManaged var launchDate: Date?
}
